.gitattributes

```diff
@@ -0,0 +1 @@
+*	-text
```

Comment:
We're only broken with `core.autocrlf=true`, right? IMO we should respect people's git configurations and only deviate when we actually have to (which is the test files only).

Probably worth setting `core.autocrlf` to true on AppVeyor to make sure we don't break under such a configuration.